### ISSUE TITLE
different subimage number obtained after running 'extract_subimages.py' for X3_sub

### ISSUE THREAD

**Issue by test-dddjjj** [jwgdmkj/LMLT#5]
After processing by running 'scripts/data_preparation/extract_subimages.py', totally obtained image number in  'datasets/DF2K/DF2K_train_LR_bicubic/X3_sub' is different from that in 'datasets/DF2K/DF2K_train_HR_sub'. So that error occurred when training launched. Yet, image numbers in 'X3_sub' and 'X4_sub' are same as that in 'HR_sub', after processing data . Are parameters below right?

# LRx2 images
    opt['input_folder'] = 'datasets/DF2K/DF2K_train_LR_bicubic/X2'
    opt['save_folder'] = 'datasets/DF2K/DF2K_train_LR_bicubic/X2_sub'
    opt['crop_size'] = 240
    opt['step'] = 120
    opt['thresh_size'] = 0
    extract_subimages(opt)

# LRx3 images
    opt['input_folder'] = 'datasets/DF2K/DF2K_train_LR_bicubic/X3'
    opt['save_folder'] = 'datasets/DF2K/DF2K_train_LR_bicubic/X3_sub'
    opt['crop_size'] = 160
    opt['step'] = 80
    opt['thresh_size'] = 0
    extract_subimages(opt)

# LRx4 images
    opt['input_folder'] = 'datasets/DF2K/DF2K_train_LR_bicubic/X4'
    opt['save_folder'] = 'datasets/DF2K/DF2K_train_LR_bicubic/X4_sub'
    opt['crop_size'] = 120
    opt['step'] = 60
    opt['thresh_size'] = 0
    extract_subimages(opt)

**Comment by jwgdmkj** [comment]
Yes, the parameters you provided are correct. When I used the code, I did not modify anything from the original code except for the directory. Could you please try running it again? For reference, the original code I used can be found at [here](https://github.com/XPixelGroup/BasicSR/blob/master/scripts/data_preparation/extract_subimages.py).

**Comment by Carry-w** [comment]
@test-dddjjj @Quyangya 哥 你能通过它的预训练模型跑测试集吗，我一直跑不起来，不知道他的测试集结构是啥样的，以及测试集放在哪里，如果你能跑，可以给我看看吗。谢谢哥。
